modules/terraform-aci-l3out-interface-profile/variables.tf

```diff
@@ -200,6 +200,7 @@ variable "interfaces" {
     ip_a            = optional(string)
     ip_b            = optional(string)
     ip_shared       = optional(string)
+    lladdr          = optional(string)
```

Comment:
This should be `optional(string, "::")` to ensure the correct value is used if not specified.